lib/portlayer/storage/vsphere/export.go

```diff
@@ -156,7 +163,10 @@ func (i *ImageStore) Export(op trace.Operation, id, ancestor string, spec *archi
 		return nil, err
 	}
 
-	return tar, nil
+	return &storage.ProxyReadCloser{
```

Comment:
It's gone. I'm not actually sure why CleanupReader wasn't getting typed as an io.Closer but the very slightly different `ProxyCloser` I tried worked, so I switched.